CONTRIBUTING.md

```diff
@@ -22,12 +22,23 @@ To declare a dependency on an `npm` package, pass `js` to the generator:
 bundle exec thor component_generator my_component_name --js=some-npm-package-name
 ```
 
+## Running and writing tests
+
+Run the whole test suite with Rake: `bundle exec rake` and only execute some tests by supplying a file glob to the test command: `TESTS="test/components/YOUR_COMPONENT_test.rb" bundle exec rake`
+
+### Inegration tests
+
+Primer ViewComponents utilizes Cuprite for integration testing. Integration testing is useful for components that rely on JavaScript functionality since Curpite runs the tests in a real browser.
+
+The integration tests run in a headless Chrome browser. Passing the `HEADLESS=false` environment variable to the test command will run the integration tests in a normal browser on the desktop.
```

Comment:
```suggestion
### System tests

Primer ViewComponents utilizes Cuprite for system testing. System testing is useful for components that rely on JavaScript functionality since Curpite runs the tests in a real browser.

The system tests run in a headless Chrome browser. Passing the `HEADLESS=false` environment variable to the test command will run the system tests in a normal browser on the desktop.
```